config/samples/saas_v1alpha1_system.yaml

```diff
@@ -117,15 +121,17 @@ spec:
       port: 25
       authProtocol: none
       opensslVerifyMode: none
-      starttlsAuto: false
+      starttls: false
     apicastAccessToken:
       fromVault:
         path: secret/data/path/system-master-apicast
         key: ACCESS_TOKEN
-    zyncAuthToken:
-      fromVault:
-        path: secret/data/path/zync
-        key: ZYNC_AUTHENTICATION_TOKEN
+    zync:
+      endpoint: https://zync:8080
```

Comment:
It's a bit confusing to have `https` on port `8080` :upside_down_face: 
But I understand this is just an example.